domains/nx.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jadfoq",
+        "email": "jadfoq@getgoogleoff.me"
+    },
+    "record": {
+    "A": "185.31.40.24"
```

Comment:
A record needs to be in an array + site is not found

Reply:
it is a alwaysdata ip https://help.alwaysdata.com/en/sites/use-external-addresses/